app/controllers/projects_controller.rb

```diff
@@ -118,7 +118,7 @@ def revision_confirmation; end
   def contents
     project
     @num_files = project.asset_count(session_id: current_user.mediaflux_session)
-    @file_list = project.file_list(session_id: current_user.mediaflux_session, size: 20)
+    @file_list = project.file_list(session_id: current_user.mediaflux_session, size: 100)
```

Comment:
Fetching 20 or 100 files takes just the same amount of time since the bulk of it is creating the HTTP request to Mediaflux. Therefore we fetch 100 files to begin with and show only the first 10 to the user and show the other 90 via JavaScript once they click the "Show More" button.